recipes/gdsfactory/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "gdsfactory" %}
+{% set version = "5.11.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gdsfactory-{{ version }}.tar.gz
+  sha256: af1bbee059a1d1cb7becbec85331a6cc899dff58d55086fa392159f93bb5b065
+
+build:
+  skip: true  # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  entry_points:
+    - gf = gdsfactory.gf:gf
```

Comment:
You have entry points and scripts upstream. Are both of these needed?

Ref:

Scripts: https://github.com/gdsfactory/gdsfactory/blob/5f5b045b442b372241ba2c2158e296f7f72afd11/setup.py#L37
Entry points: https://github.com/gdsfactory/gdsfactory/blob/5f5b045b442b372241ba2c2158e296f7f72afd11/setup.py#L46

Only entry points would be better as the package can then be noarch.

Reply:
@joamatab did you try this suggestion? noarch packages are better b/c require less rebuilds.

Reply:
yes Filipe, i added entry points as suggested by @BastianZim 

See https://github.com/conda-forge/staged-recipes/pull/19346/files

Reply:
> @joamatab did you try this suggestion? noarch packages are better b/c require less rebuilds.

@joamatab we are talking about making it `noarch` now that you have the entrypoints. You will need a with `noarch: python` above the entrypoint.

Reply:
Hi Filipe,

just added `noarch: python`

https://github.com/conda-forge/staged-recipes/pull/19346/commits/cfe2b218e4086f0dcaa866d609391d6bccf06926